### PR TITLE
Engine refactor

### PR DESCRIPTION
**Pull request by jeromesimeon**

### CODE REVIEW

File: packages/cicero-cli/package.json [hunk 1]
```diff
@@ -49,8 +49,8 @@
     "@accordproject/cicero-core": "0.10.2",
     "@accordproject/cicero-engine": "0.10.2",
     "@accordproject/cicero-tools": "0.10.2",
-    "@accordproject/ergo-compiler": "0.6.4-20190313182634",
-    "composer-concerto": "0.60.0",
+    "@accordproject/ergo-compiler": "0.6.4-20190319173619",
```
Comment (dselman) [inline packages/cicero-cli/package.json]: Should we release Ergo Compiler first and ref a stable release?

File: packages/cicero-core/package.json [hunk 2]
```diff
@@ -61,11 +61,10 @@
     "webpack-cli": "^3.1.0"
   },
   "dependencies": {
-    "@accordproject/ergo-compiler": "0.6.4-20190313182634",
+    "@accordproject/ergo-compiler": "0.6.4-20190319173619",
```
Comment (dselman) [inline packages/cicero-core/package.json]: As above.

**Comment by dselman** [comment]
The build failure is related to `axios` -- the library that Concerto uses to download CTO files from the network.

File: packages/cicero-cli/package.json (hunk 1 above)
Comment (jeromesimeon) [reply packages/cicero-cli/package.json]: Yes. That was just a way to test with the latest dev version. I'm still unwinding this branch.

**Pull request merged**